### ISSUE TITLE
Better endgame detection

### ISSUE THREAD

**Issue by skytreader** [skytreader/wherewholf#1]
Consider,

```
The village goes to sleep...
Werewolves wake up!
Night has ended and the village awakes...
The werewolves killed Chad, a Villager!
Vote now who to lynch...
Shara voted to lynch JE.
JE voted to lynch Shara.
Gab voted to lynch Shara.
Christine voted to lynch Shara.
You chose to lynch Shara, a Werewolf!
The village goes to sleep...
Werewolves wake up!
Night has ended and the village awakes...
The werewolves killed Gab, a Villager!
Vote now who to lynch...
JE voted to lynch Christine.
Christine voted to lynch Christine.
You chose to lynch Christine, a Werewolf!
The villagers won!
```

Since the Werewolf will definitely achieve parity on night 2, by then assuming that the remaining Werewolf will act rationally (i.e., don't lynch yourself!), everything should've been settled in favor of Werewolves.

This also shows that the game balance as set forth in the example right now is working ~~against Profeessor Werewolf~~ too much for the Werewolf faction...

(**EDIT:** I have no idea who I'm calling "Professor Werewolf" nor why would I think the odds are ever against them.)

**Comment by skytreader** [comment]
There is something wrong with draw detection. Consider the following trace:

```
2019-12-26 04:43:49,934 - INFO - The village goes to sleep...
2019-12-26 04:43:49,934 - INFO - Werewolves wake up!
2019-12-26 04:43:49,934 - INFO - Shara, Werewolf suggested to kill Charles
2019-12-26 04:43:49,934 - INFO - Suggestion not accepted
2019-12-26 04:43:49,934 - INFO - Christine, Werewolf suggested to kill Chad
2019-12-26 04:43:49,934 - INFO - Suggestion accepted
2019-12-26 04:43:49,934 - INFO - Night has ended and the village awakes...
2019-12-26 04:43:49,935 - INFO - The werewolves killed Chad, a Villager!
2019-12-26 04:43:49,935 - INFO - Vote now who to lynch...
2019-12-26 04:43:49,935 - INFO - The candidates for lynching are [Gab, Christine, Charles]
2019-12-26 04:43:49,935 - INFO - Shara voted to lynch Christine.
2019-12-26 04:43:49,935 - INFO - Christine voted to lynch Charles.
2019-12-26 04:43:49,935 - INFO - Gab voted to lynch Christine.
2019-12-26 04:43:49,935 - INFO - Charles voted to lynch Christine.
2019-12-26 04:43:49,935 - INFO - JE voted to lynch Charles.
2019-12-26 04:43:49,935 - INFO - You chose to lynch Christine, a Werewolf!
2019-12-26 04:43:49,935 - INFO - The village goes to sleep...
2019-12-26 04:43:49,935 - INFO - Werewolves wake up!
2019-12-26 04:43:49,935 - INFO - Shara, Werewolf suggested to kill Gab
2019-12-26 04:43:49,935 - INFO - Suggestion accepted
2019-12-26 04:43:49,935 - INFO - Night has ended and the village awakes...
2019-12-26 04:43:49,935 - INFO - The werewolves killed Gab, a Villager!
2019-12-26 04:43:49,935 - INFO - Vote now who to lynch...
2019-12-26 04:43:49,935 - INFO - The candidates for lynching are [JE, Charles]
2019-12-26 04:43:49,936 - INFO - Charles voted to lynch JE.
2019-12-26 04:43:49,936 - INFO - Shara voted to lynch JE.
2019-12-26 04:43:49,936 - INFO - JE voted to lynch Charles.
2019-12-26 04:43:49,936 - INFO - You chose to lynch JE, a Villager!
2019-12-26 04:43:49,936 - INFO - It's a draw!
```

At this point, night falls and Shara and Charles are on a 1-1 situation which would definitely settle in favor of Shara, the Werewolf.

**Comment by skytreader** [comment]
Fixed in 7230bfd45b763d04d0b0057ade8fecda6ae7a20b
